add c++ flavor to tricking the norm

kurz-m opened the pull request:
## Description
I added a small section within __Fighting the norm__ to add some C++ flavor to the C programming by using a slightly adjusted getter and setter setup. Please check and also make changes or add comments on where to change some descriptions.

Commit:
add c++ flavor to tricking the norm

haglobah commented:
Looking good!

One small remark: It took me three attempts to understand what the intention of the last sentence of your edit is. That is something you might want to rephrase.

kurz-m commented:
Yep, true!

Thinking about that sentence again, it can just be removed because the static variable within the getter is a pointer to the underlying variable and therefore, even if the value is changed, either by the setter function or e.g. strings in an array of strings, it will be still reflected in the variable returned by the getter. Correct me if I'm wrong :)

Commit:
remove unclear sentence from c++ flavor

Commit:
fix spelling within the 'all' section

kurz-m commented:
What do I need to change to also make it possible to click on the new section I added in order for the website to jump to it?

Commit:
Update header of new section
Commit updates the header of the new section to be able to click it
on the left side navbar. Solution was to remove the whitespace
at the end
